src/components/PolicyTopologyPage.tsx

```diff
@@ -452,10 +476,53 @@ const PolicyTopologyPage: React.FC = () => {
           const { nodes, edges } = parseDotToModel(dotString);
           setParseError(null);
 
+          // Separate group nodes from normal nodes
+          const groupNodes = nodes.filter((n) => n.type === 'group');
+          const normalNodes = nodes.filter((n) => n.type !== 'group');
+
+          // Dynamically generate the list of resource types
+          const uniqueTypes = Array.from(
```

Comment:
@jasonmadigan confirming, nothing special to do here when new resources are added to the topology for them to show up in the list?
I'm thinking about ServiceMonitor & PodMonitors being added in https://github.com/Kuadrant/kuadrant-operator/pull/1164

Reply:
exactly yes. they will show up fine now, but you will need to opt-in to show them by default (via `showByDefault). Shout if there's any obs resources you feel we should show by default